sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py

```diff
@@ -860,7 +863,6 @@ def start_hold_music(
         target_participant: 'CommunicationIdentifier',
         play_source: Union['FileSource', 'TextSource', 'SsmlSource'],
         *,
-        loop: bool = True,
```

Comment:
This method should be backwards compatible. Don't remove it from the method list, but drop it from the method instead.

Right now, if somebody has `loop= true` this will fail.

Reply:
@cochi2 , some reason i dont see my comment on this which I have posted yesterday. 

You mean to keep the method signature as it is and remove the loop from sending the value to api? if yes then, if any user tries to set the loop (or if somebody already set to loop) will behave as expected and end up being bug which we might want to avoid?

Reply:
That's true. But we also are marking the method as deprecated.
We want to avoid breaking production environments for Contoso.